app/adapters/publishing_adapter.rb

```diff
@@ -200,7 +200,7 @@ def save_section_content(section, manual, republish: false)
     organisation = organisation_for(manual)
 
     update_type = case version_type(republish) || section.version_type
-                  when :new, :major
+                  when :major
                     GdsApiConstants::PublishingApiV2::MAJOR_UPDATE_TYPE
                   when :minor
                     GdsApiConstants::PublishingApiV2::MINOR_UPDATE_TYPE
```

Comment:
While in principle I think this is a sensible change, at the moment there is [some duplication][1] between `PublishingAdapter#save_section_content` and `#save_manual_content`. I had it on my to-do list to remove this duplication, but hadn't got round to DRY-ing it up. I'm slightly wary of making this change which causes the duplicate code to diverge. I'd rather we made the same change to both (if that's possible).

[1]: https://github.com/alphagov/manuals-publisher/blob/8d9878843a8bb6b80405007588d652d8f6b5c05b/app/adapters/publishing_adapter.rb#L98-L107